cmd/puppet-db/root.go

```diff
@@ -120,10 +125,11 @@ func setCmdFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringP(
 		"config",
 		"c",
-		getDefaultConfig(),
+		"",
```

Comment:
Should use default value: `filepath.Join(usr.HomeDir, ".puppetlabs", "client-tools", "puppetdb.conf")`